pkg/sql/logictest/testdata/logic_test/drop_table

```diff
@@ -25,6 +25,11 @@ SELECT * FROM a
 statement ok
 DROP TABLE a
 
+query T
+SELECT status FROM [SHOW JOBS]
+----
+running (waiting for GC TTL)
```

Comment:
I think we should change this to not have "running" in the status. it's better to just see "waiting for GC TTL"